<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shortcut_dialog</class>
 <widget class="QDialog" name="shortcut_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcuts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QGroupBox" name="main_window_group_box">
       <property name="title">
        <string>Main Window Shortcuts</string>
       </property>
       <layout class="QVBoxLayout" name="main_window_layout">
        <item>
         <spacer name="verticalSpacer_main_window">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="game_window_group_box">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Game Window Shortcuts</string>
       </property>
       <layout class="QVBoxLayout" name="game_window_layout">
        <item>
         <spacer name="verticalSpacer_game_window">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
